streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/AccessTokenServiceImpl.java

```diff
@@ -116,14 +117,22 @@ public RestResponse toggleToken(Long tokenId) {
             ? AccessToken.STATUS_DISABLE
             : AccessToken.STATUS_ENABLE;
 
-    AccessToken updateObj = new AccessToken();
-    updateObj.setStatus(status);
-    updateObj.setId(tokenId);
-    return RestResponse.success(this.updateById(updateObj));
+    tokenInfo.setStatus(status);
+    return RestResponse.success(this.updateById(tokenInfo));
   }
 
   @Override
   public AccessToken getByUserId(Long userId) {
     return baseMapper.selectByUserId(userId);
   }
+
+  @Override
+  public boolean updateById(AccessToken entity) {
```

Comment:
I noticed that there are many similar writing methods. Can it be optimized through templates or options?

Reply:
I do not understand